pkg/controllers/clusterset/syncclusterrolebinding/syncclusterrolebinding_controller.go

```diff
@@ -50,15 +49,10 @@ func NewReconciler(kubeClient kubernetes.Interface,
 
 // Run a routine to sync the clusterrolebinding periodically.
```

Comment:
correct the comments?

Reply:
please check the `pkg/controllers/gc/garbagecollector_controller.go Run()` as well. no need go routine.